server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -254,6 +255,9 @@ function insertConsent($reviewConsent, $virtualConsent, $signature, $appointment
 		throw new Exception("There was an issue on the server. Please refresh the page and try again.", MY_EXCEPTION);
 	}
 }
+function validateConsent($virtualConsent, $signature, $appointmentId) {
```

Comment:
nit: maybe consider calling this `hasConsentedToVirtualPreparation` or `consentedToVirtualPreparation`, etc. I would expect a `validate` method to just check inputs and throw an error if something is amiss